docker/quickstart/docker-compose.yml

```diff
@@ -15,19 +15,22 @@ services:
       - "3306:3306"
     volumes:
       - ../mysql/init.sql:/docker-entrypoint-initdb.d/init.sql
+      - ${DATA_STORAGE_FOLDER}/mysql:/var/lib/mysql
 
   zookeeper:
-    image: confluentinc/cp-zookeeper:5.2.1
+    image: confluentinc/cp-zookeeper:5.4.0
     hostname: zookeeper
     container_name: zookeeper
     ports:
       - "2181:2181"
     environment:
       ZOOKEEPER_CLIENT_PORT: 2181
       ZOOKEEPER_TICK_TIME: 2000
+    volumes:
+      - ${DATA_STORAGE_FOLDER}/zookeeper:/var/opt/zookeeper
```

Comment:
Not super familiar with Docker volumes and have a question. After a `docker-compose down`, will I be able to get rid of the data and start with a clean slate? Or, do I need to manually delete the data under `/tmp/datahub`?

Reply:
You will need to manually remove the data under the `${DATA_STORAGE_FOLDER}`. Also, you can delete only one of the subfolders if you only want to restore their state (_i.e removing only the zookeeper folder_).